config/kubernetes-sigs/sig-cluster-lifecycle/teams.yaml

```diff
@@ -87,6 +88,8 @@ teams:
     - CecileRobertMichon
     - cpanato
     - devigned
+    - jackfrancis
+    - jsturtevant
     - juan-lee
```

Comment:
can you please also remove juan-lee here and nader-ziada above? ref https://github.com/kubernetes-sigs/cluster-api-provider-azure/commit/ccd08ea27a3f20cbf49ed2b1ecdf6744b3671c93

Reply:
should I add myself to cluster-api-provider-azure-admins as well?

Reply:
yes

Reply:
done